ui/components/multichain/account-list-menu/account-list-menu.tsx

```diff
@@ -303,6 +305,22 @@ export const AccountListMenu = ({
     }
   }
 
+  const handleTraceFromTabName = (tabName: string) => {
+    switch (tabName) {
+      case 'nfts':
+        endTrace({ name: TraceName.AccountOverviewNftsTab });
+        trace({ name: TraceName.AccountOverviewNftsTab });
+        break;
+      case 'activity':
+        endTrace({ name: TraceName.AccountOverviewActivityTab });
+        trace({ name: TraceName.AccountOverviewActivityTab });
+        break;
+      default:
+        endTrace({ name: TraceName.AccountOverviewAssetListTab });
+        trace({ name: TraceName.AccountOverviewAssetListTab });
+    }
+  };
+
```

Comment:
Nit: can we structure this `switch` and also the one in `account-overview-tabs.tsx` as a shared function, and with a dictionary instead of a `switch`?  The goal is DRY.

Reply:
Good point! Applied here: https://github.com/MetaMask/metamask-extension/pull/28086/commits/a5c2ae2d963f267cc663d9e2fcee462e82e32bed